bin/build_container_image

```diff
@@ -3,7 +3,8 @@
 REGISTRY=${REGISTRY:-"docker.io"}
 ORGANIZATION=${ORGANIZATION:-"manageiq"}
 DEFAULT_TAG="latest"
-[[ $TRAVIS_BRANCH != "master" ]] && DEFAULT_TAG="$DEFAULT_TAG-$TRAVIS_BRANCH"
+[ "$GITHUB_EVENT_NAME" == "push" -a "$GITHUB_REF" != "refs/heads/master" ] && \
+  DEFAULT_TAG="$DEFAULT_TAG-${GITHUB_REF_NAME/\//-}"
```

Comment:
```bash
$ GITHUB_EVENT_NAME=push GITHUB_REF_NAME="master" bin/build_container_image
docker.io/manageiq/rpm_build:latest-master
$ GITHUB_EVENT_NAME=push GITHUB_REF_NAME="foobar" bin/build_container_image
docker.io/manageiq/rpm_build:latest-foobar
$ GITHUB_EVENT_NAME=push GITHUB_REF_NAME="feature/foobar" bin/build_container_image
docker.io/manageiq/rpm_build:latest-feature-foobar
$ bin/build_container_image
docker.io/manageiq/rpm_build:latest
```

Reply:
What do you think of switching to `latest-master`so that we can get rid of this conditional logic?

Reply:
It's super common to have latest be the latest, so I'm hesitant to _not_ use latest now that we've already started.